Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "bins",
     "cli",
     "kernel",
+    "server"
```

Comment:
Explicitly adding `server` to the workspace. It's already included via path dependency but for clarity's sake I think it was worth it to introduce it here.